packages/belos/src/BelosGmresPolySolMgr.hpp

```diff
@@ -453,54 +383,24 @@ GmresPolySolMgr<ScalarType,MV,OP>::getValidParameters() const
-    pl->set("Maximum Restarts", static_cast<int>(maxRestarts_default_),
-      "The maximum number of restarts allowed for each\n"
-      "set of RHS solved.");
-    pl->set("Maximum Iterations", static_cast<int>(maxIters_default_),
-      "The maximum number of block iterations allowed for each\n"
-      "set of RHS solved.");
-    pl->set("Num Blocks", static_cast<int>(numBlocks_default_),
-      "The maximum number of blocks allowed in the Krylov subspace\n"
-      "for each set of RHS solved.");
-    pl->set("Block Size", static_cast<int>(blockSize_default_),
-      "The number of vectors in each block.  This number times the\n"
-      "number of blocks is the total Krylov subspace dimension.");
+    pl->set("Outer Solver", static_cast<const char *>(outerSolverType_default_),
+      "The outer solver that this polynomial is used to precondition.");
     pl->set("Verbosity", static_cast<int>(verbosity_default_),
       "What type(s) of solver information should be outputted\n"
       "to the output stream.");
-    pl->set("Output Style", static_cast<int>(outputStyle_default_),
-      "What style is used for the solver information outputted\n"
-      "to the output stream.");
-    pl->set("Output Frequency", static_cast<int>(outputFreq_default_),
-      "How often convergence information should be outputted\n"
-      "to the output stream.");
     pl->set("Output Stream", Teuchos::rcp(outputStream_default_,false),
       "A reference-counted pointer to the output stream where all\n"
       "solver output is sent.");
-    pl->set("Strict Convergence", static_cast<bool>(strictConvTol_default_),
-      "After polynomial is applied, whether solver should try to achieve\n"
-      "the relative residual tolerance.");
-    pl->set("Show Maximum Residual Norm Only", static_cast<bool>(showMaxResNormOnly_default_),
-      "When convergence information is printed, only show the maximum\n"
-      "relative residual norm when the block size is greater than one.");
-    pl->set("Implicit Residual Scaling", static_cast<const char *>(impResScale_default_),
-      "The type of scaling used in the implicit residual convergence test.");
-    pl->set("Explicit Residual Scaling", static_cast<const char *>(expResScale_default_),
-      "The type of scaling used in the explicit residual convergence test.");
     pl->set("Timer Label", static_cast<const char *>(label_default_),
       "The string to use as a prefix for the timer labels.");
     pl->set("Orthogonalization", static_cast<const char *>(orthoType_default_),
-      "The type of orthogonalization to use: DGKS, ICGS, or IMGS.");
-    pl->set("Orthogonalization Constant",static_cast<MagnitudeType>(DefaultSolverParameters::orthoKappa),
-      "The constant used by DGKS orthogonalization to determine\n"
-      "whether another step of classical Gram-Schmidt is necessary.");
+      "The type of orthogonalization to use to generate polynomial: DGKS, ICGS, or IMGS.");
```

Comment:
Omit the colon and everything after it.  (The OrthoManagerFactory is responsible for the set of valid OrthoManager types.)

Reply:
See above.

Reply:
I'm not suggesting changing the default behavior here, just removing duplicated code.